packages/ra-ui-materialui/src/button/BulkExportButton.tsx

```diff
@@ -85,7 +85,7 @@ interface Props {
     label?: string;
     onClick?: (e: Event) => void;
     selectedIds: Identifier[];
-    resource?: string;
+    resource: string;
```

Comment:
why? The `resource` is usually injected by the parent component, and we don't document the need to add it manually. This is a breaking change.

Reply:
I was using type-script with strictMode = true. I'll change it.